src/App.tsx

```diff
@@ -12,161 +18,45 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={({ isActive }) => cn('navbar-item', {
+              'has-background-grey-lighter': isActive,
+            })}
+            to="/"
+          >
             Home
-          </a>
+          </NavLink>
```

Comment:
you have some repeated code for the links in `NavBar`, let's move it to the separate component